beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestHardenCatalogPlugin.java

```diff
@@ -248,7 +249,7 @@ public void addCatalogVersion(final String catalogResource) throws Exception {
             final StandaloneCatalogWithPriceOverride inputCatalogVersionWithOverride = new StandaloneCatalogWithPriceOverride(inputCatalogVersion, priceOverride, internalTenantContext.getTenantRecordId(), internalCallContextFactory);
 
             if (versionedCatalog == null) {
-                versionedCatalog = new DefaultVersionedCatalog(clock);
+                versionedCatalog = new DefaultVersionedCatalog(new ArrayList<>());
```

Comment:
You could use the no-arg constructor instead.

Reply:
Will remove the CTOR with the array -- it is now useless.